bluemira/geometry/coordinates.py

```diff
@@ -888,13 +888,26 @@ def normal_vector(side_vectors: np.ndarray) -> np.ndarray:
 
     Parameters
     ----------
-    side_vectors:
+    side_vectors: np.ndarray
```

Comment:
We don't type in docstrings on bluemira. Only on the function definition. The types are automatically copied over to the documentation docstrings when we build them. Please could you remove them

Reply:
Done!